Source/Particles/LaserParticleContainer.cpp

```diff
@@ -607,6 +621,8 @@ LaserParticleContainer::PostRestart ()
 {
     if (!m_enabled) return;
 
+    m_laser_injection_box= Geom(0).ProbDomain();
```

Comment:
Because we are not calling `InitData` anymore, I think that we do not need this line anymore.
I will remove it.